packages/router-generator/src/generator.ts

```diff
@@ -306,7 +306,13 @@ export async function generator(config: Config) {
 
       if (replaced !== routeCode) {
         logger.log(`🟡 Updating ${node.fullPath}`)
-        await fsp.writeFile(node.fullPath, replaced)
+        const formattedReplaced = await prettier.format(replaced, {
```

Comment:
👍🏼

Reply:
do we really want to format the route files? why?

Reply:
@schiller-manuel 

I prefer to format the route files because they are part of my code and affect my code style.
To have full control over formatting, I think we need to separate the config, considering Sean Cassiere's opinion in the previous comment. Otherwise, the whole pull request is pointless. ty

Reply:
I don't think router-generator should format route files. This are implementation files that the user has full control over when it comes to code formatting, so I'd say it is the user's duty to setup the desired codeformatting independently from the generator.